src/main.rs

```diff
@@ -126,6 +126,23 @@ fn main() -> Result<()> {
         exit(0);
     }
 
+    if matches.is_present(Other_commands::whoami) {
+        if let Some(_value) = matches.subcommand_matches("token") {
+            openid::print_token(&context);
```

Comment:
I think what you look for is : 
```suggestion
        if matches.is_present("token") {
            openid::print_token(&context);
```